src/foraging_gui/Foraging.py

```diff
@@ -264,7 +264,7 @@ def _revert_to_previous_parameters(self):
         parameters={}
         for attr_name in dir(self):
             if attr_name.startswith('WarmupBackup_') and attr_name!='WarmupBackup_' and attr_name!='WarmupBackup_warmup':
-                parameters[attr_name[3:]]=getattr(self,attr_name)
+                parameters[attr_name[13:]]=getattr(self,attr_name)
```

Comment:
Useing `attr_name.strip('WarmupBackup_')` is better than hardcoding `13` here. Otherwise the code breaks if one change the prefix.

Reply:
Done!

Reply:
We can't use `attr_name.strip('WarmupBackup_')` here. It should be attr_name.replace('WarmupBackup_','') instead. See below:
 
![image](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/assets/109394934/9bb2711f-1a14-4f1d-90ef-b86521df121c)

Reply:
Oh yes, my bad.